cognitedata/reveal
# fix: general cylinder arc angle is too big sometimes

#### Type of change

![Bug](https://img.shields.io/badge/Type-Bug-red) 

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/BND3D-1780

## Description :pencil:


The arc angle computation in the cylinder- and circle shaders when `atan` returns a negative angle `theta` such that `theta + 2 * PI < angle`. `angle` is the base angle from which `arcAngle` starts in the shader.

## How has this been tested? :mag:



- Download this zip, unzip it in `examples/public`

[angles.zip](https://github.com/cognitedata/reveal/files/10899803/angles.zip)

- Open https://localhost:3000/?modelUrl=sing
- In master, observe that a couple of the pillars have an overly drawn cylinder
- In this branch, observe that not.

## Test instructions :information_source:




## Checklist :ballot_box_with_check:
